docs/source/user-manual/index.md

```diff
@@ -0,0 +1,19 @@
+---
+myst:
+  html_meta:
+    "description": "This is the end-User documentation for Plone 6 Volto frontend."
+    "property=og:description": "Volto is the React-based frontend for the Plone CMS. It is the default UI for the Plone 6 release."
+    "property=og:title": "Frontend"
+    "keywords": "Volto, Plone, frontend, React, User manual"
+---
+
+# User Manual
```

Comment:
It is helpful to use labels, which allow authors to reference the label from arbitrary locations in the rest of the docs with the [`ref` directive](https://myst-parser.readthedocs.io/en/latest/syntax/syntax.html#targets-and-cross-referencing).

```suggestion

(user-manual-label)=

# User Manual
```

Reply:
sorry I may have forgotten to add it here.